test/notebooks/nbtest_transformers.py

```diff
@@ -0,0 +1,43 @@
+import os
+import textwrap
+import pytest
+
+from notebook_test_utils import (access_to_temp_secret_store, notebook_runner, uploading_hack)
+
+
+@pytest.mark.parametrize(
+    "notebook_file",
+    [
+        'sequence_classification.ipynb',
+        'question_answering.ipynb',
+        'masked_modelling.ipynb',
+        'token_classification.ipynb',
+        'text_generation.ipynb',
+        pytest.param('translation.ipynb', marks=pytest.mark.xfail(
+            reason='waiting for the Sacremoses tokenizer to be installed in the SLC')),
+        'zero_shot_classification.ipynb'
+    ]
+)
+def test_transformers(notebook_runner, uploading_hack, notebook_file) -> None:
+
+    running_hack = (
+        'running_model',
+        textwrap.dedent("""
+            errors = %sql --with udf_output SELECT error_message FROM udf_output where error_message != 'None'
+            assert not errors, str(errors)
+        """)
+    )
+    running_pyx_hack = (
+        'running_model_pyx',
```

Comment:
```suggestion
        'running_model_pyexasol',
```

reason, pyx sounds too much like cython files or some special python file extension.